charts/mysql/values.yaml

```diff
@@ -0,0 +1,82 @@
+# Default values for mysql containers, can be overwritten at build time by environment configurations
+
+global:
+  # global.storageClass -- Storage class to use when provisioning persistent disks
+  storageClass: standard
+  # global.applicationVersion -- What version of mysql to deploy
+  applicationVersion: latest
+
+# name -- A name for the statefulset that will be substituted into resource definitions
```

Comment:
I know we don't have a standard when it comes to organizing the values file. However, most of them have deployment values under a deployment key. Can you do the same here?

Reply:
Yes! Thanks for catching this. I'm actually going to move this to a new ticket and then poke you on this b/c I have to update the chart, which would blow up this PR.